setup.py

```diff
@@ -19,13 +19,12 @@
     install_requires=[
         # fixed versions.
         'fastavro>=1.5.1,<2.0.0',
-        'pandas>=1.1',
-        # https://pandas.pydata.org/pandas-docs/version/1.1/getting_started/install.html#dependencies
-        'numpy>=1.15.4',
+        'pandas>=2.0,<3.0.0',
+        'numpy>=2.0',
```

Comment:
So that it does not break everything again on next numpy major
```suggestion
        'numpy>=2.0,<3.0.0',
```